docs/wordpress.md

```diff
@@ -22,116 +22,47 @@ with Docker containers. This quick-start guide demonstrates how to use Compose t
-
-6. Create a `wp-config.php` file within the `wordpress` directory.
-
-    A supporting file is needed to get this working. At the top level of the wordpress directory, add a new file called `wp-config.php` as shown. This is the standard WordPress config file with a single change to point the database configuration at the `db` container:
-
-        <?php
-        define('DB_NAME', 'wordpress');
-        define('DB_USER', 'wordpress');
-        define('DB_PASSWORD', 'wordpress');
-        define('DB_HOST', "db:3306");
-        define('DB_CHARSET', 'utf8');
-        define('DB_COLLATE', '');
-
-        define('AUTH_KEY',         'put your unique phrase here');
-        define('SECURE_AUTH_KEY',  'put your unique phrase here');
-        define('LOGGED_IN_KEY',    'put your unique phrase here');
-        define('NONCE_KEY',        'put your unique phrase here');
-        define('AUTH_SALT',        'put your unique phrase here');
-        define('SECURE_AUTH_SALT', 'put your unique phrase here');
-        define('LOGGED_IN_SALT',   'put your unique phrase here');
-        define('NONCE_SALT',       'put your unique phrase here');
-
-        $table_prefix  = 'wp_';
-        define('WPLANG', '');
-        define('WP_DEBUG', false);
-
-        if ( !defined('ABSPATH') )
-          define('ABSPATH', dirname(__FILE__) . '/');
-
-        require_once(ABSPATH . 'wp-settings.php');
-        ?>
-
-7. Verify the contents and structure of your project directory.
-<!--
-        Dockerfile
-        docker-compose.yaml
-        wordpress/
-          index.php
-          license.txt
-          readme.html
-          wp-activate.php
-          wp-admin/
-          wp-blog-header.php
-          wp-comments-post.php
-          wp-config-sample.php
-          wp-config.php
-          wp-content/
-          wp-cron.php
-          wp-includes/
-          wp-links-opml.php
-          wp-load.php
-          wp-login.php
-          wp-mail.php
-          wp-settings.php
-          wp-signup.php
-          wp-trackback.php
-          xmlrpc.php
-      -->
-
-    ![WordPress files](images/wordpress-files.png)
+          wordpress:
+            depends_on:
+              - db
+            image: wordpress:latest
+            links:
+              - db
+            ports:
+              - "8000:80"
+            restart: always
+            environment:
+              WORDPRESS_DB_HOST: db:3306
+              WORDPRESS_DB_PASSWORD: wordpress
+
+    ***NOTE***: You can also use the drop in replacement `MariaDB` by simply replacing with the official `mariadb:10`.
 
 ### Build the project
 
-With those four new files in place, run `docker-compose up` from your project directory. This will pull and build the needed images, and then start the web and database containers.
+With those four new files in place, run `docker-compose up -d` from your project directory. This will pull and build the needed images, and then start the wordpress and database containers.
```

Comment:
@dnephin something seems to be missing here, because as written:
* this doesn't make sense anymore "with those four new files in place" (which 4 files? because now we don't tell them to create them)
*  it doesn't run

I see that the `docker-compose.yml` file is calling different DB and Wordpress images, defining different volumes, wordpress, etc. but how is the .yml supposed to find what it needs?  Doesn't seem to be working for me.

Also there doesn't seem to be a reference point for `MariaDB` comment. Where would I see that referenced? In one of the containers? In another config file?